script.js

```diff
@@ -1,4 +1,309 @@
+//Add global states to include wins and draws
+var playerScore = 0;
+var comScore = 0;
+var drawScore = 0;
+
+//Add states for username and game mode
+var userName = "";
+var gameMode = "";
+
+//Game mode checker
+var gameModeState = 0;
+
+//Add WinState for korean mode
+var playerLastWinState = 0;
+var comLastWinState = 0;
+var winnerState = 0;
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  //Convert input to lowercase and remove white space, allow for minor typos
+  var convertInput = input.toLowerCase().replace(/\s/g, "");
```

Comment:
Great initiative in trying to use regex to catch for white space! 
There is a [trim() function](https://www.w3schools.com/jsref/jsref_trim_string.asp) that does this too, if you want to explore